lib/common/util.js

```diff
@@ -124,7 +124,7 @@ util.arrayize = arrayize;
  */
 function format(template, args) {
   return template.replace(/{([^}]*)}/g, function(match, key) {
-    return args[key] || match;
+    return key in args ? args[key] : match;
```

Comment:
That was a good catch, now I see what you mean. Thank you for the clarification!